src/components/layout/Footer.tsx

```diff
@@ -9,6 +9,56 @@ import "./Footer.css";
 
 const Footer = () => (
   <div className="footer">
+    <div>
+      <h4 className="footer__sectionHeader">Citing Tree of Science</h4>
+      <p>
+        If Tree of Science contributes to a project that leads to a scientific
+        publication, please acknowledge this fact by citing:
+        <ul className="footer__sectionLinks citations">
+          <li>
+            <a
+              target="blank"
+              rel="noopener noreferrer"
+              href="https://revistas.unal.edu.co/index.php/ingeinv/article/view/77718/0"
+            >
+              {/* Valencia-Hernandez, D. S., Robledo, S., Pinilla, R., Duque-Méndez,
+              N. D., & Olivar-Tost, G. (2020). Algoritmo SAP para análisis de
+              citaciones: Una mejora al Árbol de la Ciencia. Ingeniería E
+              Investigación, 40(1), 45-49. */}
+              Algoritmo SAP para análisis de citaciones: Una mejora al Árbol de
+              la Ciencia
+            </a>
+          </li>
+          <li>
+            <a
```

Comment:
We might want to use the `<Reference /> component here instead, use the simple version.